src/devices/SDLJoypad/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2016 iCub Facility, Istituto Italiano di Tecnologia
+# Authors: Andrea Ruzzenenti
+# email:   andrea.ruzzenenti@iit.it
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+
+yarp_prepare_plugin(SDLJoypad
+                    CATEGORY             device
+                    TYPE                 yarp::dev::SDLJoypad
+                    INCLUDE              SDLJoypad.h
+                    EXTRA_CONFIG WRAPPER=JoypadControlServer
+                    DEPENDS              "CREATE_DEVICE_LIBRARY_MODULES;YARP_USE_SDL")
```

Comment:
there is a pratical reason or it's an estetical advice? because for me is far more readable aligned like this.. anyway if you'll insist i'll change it.. XD